src/Facades/Notification.php

```diff
@@ -0,0 +1,13 @@
+<?php
+
+namespace Native\Laravel\Facades;
+
+use Illuminate\Support\Facades\Facade;
+
```

Comment:
Add notification methods into docblock!

Reply:
updated

Reply:
Just `Object_` is a problem!
I fix it after this PR